source/interact-data/specify-query.txt

```diff
@@ -0,0 +1,476 @@
+
+  # Creates a simple query
+  Band.where(name: "Deftones")
+  
+  # Returns a Criteria object
+  # => #<Mongoid::Criteria
+  #   selector: {"name"=>"Deftones"}
+  #   options:  {}
+  #   class:    Band
+  #   embedded: false>
+
+  # Evaluate the query by converting to JSON
+  Band.where(name: "Deftones").to_json
+  
+  # Returns matching documents
+  # => [{"_id":"...","name":"Deftones"}]
+
+You can use methods such as ``first()`` and ``last()`` to return
+individual documents. You can also iterate a ``Criteria`` object by using
+methods such as ``each()`` or ``map()`` to retrieve documents from the
+server. You can use ``to_json()`` to convert a ``Criteria`` object to
+JSON.
+
+.. tip::
+
+   If you call more query methods on an existing ``Criteria`` object,
+   {+odm+} merges the filter criteria.
+
+Condition Syntax
+----------------
+
+This section describes the syntax patterns that you can use to create
+filter criteria. You can specify queries in {+odm+} by using any of the
+following syntax patterns:
+
+- Field syntax
+- Query API syntax
+- Symbol operator syntax
+
+.. note:: Query Syntax Behavior
+    
+   These syntaxes support querying embedded documents by using dot notation.
+   The syntaxes also respect :ref:`field aliases <mongoid-query-field-alias>`
+   and field types, if the field being queried is defined in the model class.
+
+The examples in this section use the following model definition:
+
+.. code-block:: ruby
+
+   class Band
+     include Mongoid::Document
+ 
+     field :name, type: String
+     field :founded, type: Integer
+     field :m, as: :member_count, type: Integer
+ 
+     embeds_one :manager
+   end
+ 
+   class Manager
+     include Mongoid::Document
+ 
+     embedded_in :band
+ 
+     field :name, type: String
+   end
+
+Field Syntax
+~~~~~~~~~~~~
+
+The field querying syntax uses the basic {+language+} hashes. The keys
+can be symbols or strings and correspond to field names in MongoDB
+documents.
+
+The following code shows two equivalent queries that use field querying
+syntax to retrieve documents in which the ``name`` field value is
+``'Depeche Mode'``:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-simple-field-query
+   :end-before: end-simple-field-query
+   :language: ruby
+   :dedent:
+
+Query API Syntax
+~~~~~~~~~~~~~~~~
+
+You can specify a Query API operator on any field by using the hash
+syntax, as shown by the following equivalent queries:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-query-api-query
+   :end-before: end-query-api-query
+   :language: ruby
+   :dedent:
+
+Symbol Operator Syntax
+~~~~~~~~~~~~~~~~~~~~~~
+
+You can specify Query API operators as methods on symbols for the
+respective field name, as shown in the following code:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-symbol-query
+   :end-before: end-symbol-query
+   :language: ruby
+   :dedent:
+
+Fields
+------
```

Comment:
S: Fields seems a little to general of a title for this section
```suggestion
Field Types
------------
```
or to be more specific:
```suggestion
Query on Different Field Types
-------------------------------
```